pyathena/connection.py

```diff
@@ -136,8 +138,13 @@ def __init__(
                 profile_name=self.profile_name,
                 **self._session_kwargs,
             )
+
+        self.config: Optional[config.Config] = config
+        self.config.user_agent_extra = (
+            "PyAthena/" + pyathena.__version__ + " " + str(config.user_agent_extra or "")
```

Comment:
This looks like the user agent would end in a space when there was no extra info, or is there some stripping happening later that was already there from before?

Reply:
I totally missed that space: Nice spotted! 
I just fixed it.
Thanks for the feedback!